Add create_sysin.py

Degilio opened the pull request:
Added a new file and documentation on a routine that creates a sysin file in USS that can be used by ZOAU's mvscmd. Every time I need to use mvscmd I would have to create a routine that put the file out. This will do this via import and has additional support for multiple code pages...

Degilio committed:
Simplified code

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Degilio committed:
Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Degilio committed:
Fixed Spelling Error

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Degilio committed:
Fixed Spelling errors and removed unnecessary code
Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Degilio committed:
Added code to create a sysin file

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

luisamarti commented on pyzkiln_core/zutils/create_sysin.py:
```diff
@@ -0,0 +1,54 @@
+    print(f"Input written to: {filename}")
+
+
+if __name__ == "__main__":
```
For this new file, we need to add it to the include list in pyzkiln/pyzkiln_core/zutils/__init__.py
Also, please include the brief documentation in pyzkiln/docs/index.md
Thank you.

Degilio replied:
Done

luisamarti reviewed (changes requested):
Can you please add the missing include

Degilio committed:
Updated documentation

Singed-off-by: Frank De Gilio degilio@us.ibm.com
Fixed the doc and updated init.py

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>

Degilio committed:
Updated to fix comments by Luisa

Signed-off-by: Frank De Gilio <degilio@us.ibm.com>
